vars/filebeat.groovy

```diff
@@ -36,7 +36,13 @@ def call(Map args = [:], Closure body) {
 }
 
 def start(Map args = [:]) {
-  def output = args.containsKey('output') ? args.output : 'docker_logs.log'
+  def sanitizedOutput = ''
+  if (args.containsKey('output')) {
+    sanitizedOutput = sanitizeOutputFileName(args.output)
+  }
+  sanitizedOutput = sanitizedOutput?.trim() ? sanitizedOutput : 'docker_logs.log'
```

Comment:
this is something we make in several places, maybe worth making a step to sanitized strings (follow up)